src/python/pants/python/python_setup.py

```diff
@@ -131,9 +131,15 @@ def register_options(cls, register):
         register(
             "--resolver-jobs",
             type=int,
-            default=None,
+            default=2,
             advanced=True,
-            help="The maximum number of concurrent jobs to resolve wheels with.",
+            help=(
+                 "The maximum number of concurrent jobs to resolve wheels with. Because Pants "
+                 "can run multiple subprocesses in parallel, the total parallelism will be "
+                 "`--process-execution-local-parallelism x --python-setup-resolver-jobs`. Setting "
+                 "this option higher may result in better parallelism, but, if set too high, may "
+                 "result in starvation and Out of Memory errors."
```

Comment:
Feedback welcomed.

Reply:
Seems fine, but I'm continually worried about piling on words when the end result leaves potential misinformation. There are two small deceptions I think:
1. This assumes local, but that's not the only problem, so maybe `--process-execution-*-parallelism`
2. The word resolve gives the wrong impression since - I think - most folks think of that as downloading and we only do that in 1 process. Its wheel building and installing that use the paralellism and that makes more sense. Maybe then s/resolve wheels with/build wheels with/.